README.md

```diff
@@ -3,58 +3,100 @@
 Hubstats is a rails plugin which allows you to search and monitor pull requests made across a collection of repositories. It also gives extra statistics about users and pull requests not found on GitHub.
 
 ## Setup
+The following setup is designed to only be used when integrating this plugin into a rails application.
 
-The following setup is designed to only be used when integrating this plugin into a rails application or for when adding new migrations.
+ To add a new migration: `rails generate migration <migration-name>`
```

Comment:
It's not. I'm not really sure where to put it in the readme. I'm not even sure if it's necessary to include in the readme..... I'll move it out for now.